server/core/src/main/java/org/infinispan/server/core/telemetry/TelemetryServiceFactory.java

```diff
@@ -40,8 +53,8 @@ public Object construct(String name) {
          log.telemetryLoaded(openTelemetry);
```

Comment:
Side note: we should only log when OpenTelemetry is enabled, not disabled. Also it should use the `org.infinispan.SERVER` category.

Reply:
done the change thank you